internal/testrunner/runners/system/servicedeployer/terraform.go

```diff
@@ -42,6 +46,38 @@ type TerraformServiceDeployer struct {
 	definitionsDir string
 }
 
+// addTerraformOutputs method reads the terraform outputs generated in the json format and
+// adds them to the custom properties of ServiceContext and can be used in the handlebars template
+// like `{{TF_OUTPUT_queue_url}}` where `queue_url` is the output configured
+func addTerraformOutputs(outCtxt ServiceContext) error {
+	// Read the `output.json` file where terraform outputs are generated
+	content, err := os.ReadFile(outCtxt.OutputDir + terraformOutputJsonFile)
```

Comment:
It's better to use filepath.Join to ensure the paths are generated correctly.

```suggestion
	terraformOutputFile := filepath.Join(outCtxt.OutputDir, terraformOutputJsonFile)
	content, err := os.ReadFile(terraformOutputFile)
```

I think doing so, maybe this variable `terraformOutputJsonFile` needs to remove the `/` at the beginning (to be tested):
```
    terraformOutputJsonFile = "tfOutputValues.json"
```